static/_redirects

```diff
@@ -46,16 +46,16 @@
 /guides/traefik-ingress.html https://0-20-0.docs.pomerium.com/docs/guides/traefik-ingress 301!
 /docs/reference/forward-auth https://0-20-0.docs.pomerium.com/docs/reference/forward-auth 301!
 https://0-20-0.docs.pomerium.com/category/guides https://0-20-0.docs.pomerium.com/docs/guides
-/docs/reference/policy/allowed-domains https://0-14-0.docs.pomerium.com/reference
-/docs/reference/policy/allowed-idp-claims https://0-14-0.docs.pomerium.com/reference
-/docs/reference/policy/allowed-users https://0-14-0.docs.pomerium.com/reference
+/docs/reference/policy/allowed-domains https://0-14-0.docs.pomerium.com/reference/#allowed-domains
+/docs/reference/policy/allowed-idp-claims https://0-14-0.docs.pomerium.com/reference/#allowed-idp-claims
+/docs/reference/policy/allowed-groups https://0-14-0.docs.pomerium.com/reference/#allowed-groups
```

Comment:
I'm not sure whether it's worth adding this one (v0.14 was a long time ago), but I thought it might make sense to be consistent with the others. (And if we have them, I figured it made sense to add anchor tags so they link to the relevant section heading.)

Reply:
I agree for the sake of consistency, too. It can't hurt.